src/sass/blocks/contacts/_contacts.sass

```diff
@@ -91,5 +91,4 @@
     &__map 
         width: 100%
         height: 100%
-        background: url('./map.jpg') center center/cover no-repeat
```

Comment:
resourse deleted, but it you delete the picture from the repository ?

Reply:
+